### ISSUE TITLE
role stops working from 27.1 release on

### ISSUE THREAD

**Issue by zerwes** [Rosa-Luxemburgstiftung-Berlin/ansible-opnsense-update#4]
```
FAILED! => {"msg": "Version comparison failed: '<' not supported between instances of 'int' and 'str'"}
```

**Comment by zerwes** [comment]
problem: the facts reported have changed the syntax
new notation used in opnsense:
`"product_version": "22.7_4"`
old notation:
`"product_version": "21.10.3"`

**Comment by zerwes** [comment]
and https://github.com/Rosa-Luxemburgstiftung-Berlin/ansible-opnsense-facts just reports the version as it is notated in opnsense ...

**Comment by zerwes** [comment]
fix is no fix, 22.7_4 means 22.7 (r4) and not 22.7.4 (this is in the future!)
